lib/src/components/NxSidebarNavigation/NxSidebarNavitation.scss

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-width__open: 250px;
+$nx-sidebar-width__closed: 69px;
+$nx-sidebar-logo-height: 33px;
+$nx-sidebar-nav-background-color: $nx-blue-1000;
+$nx-sidebar-nav-text-color: $nx-white;
+$nx-sidebar-nav-help-color: $nx-grey-50;
+$nx-sidebar-nav-hover-color: $nx-indigo-600;
+$nx-sidebar-nav-selected-color: $nx-indigo-800;
+$nx-sidebar-separator-color: $nx-grey-900;
+
+
+.nx-sidebar-nav {
+  @include container-vertical;
+
+  box-sizing: border-box;
+  background-color: $nx-sidebar-nav-background-color;
+  color: $nx-sidebar-nav-text-color;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
```

Comment:
> I think a better way to do this would be to use `align-items: stretch` on `.nx-page--sidebar`

No longer relevant as of b9b047801218fedc6d13234b0795b2c66b9c2238